docs/deployments/free5gc-ulcl/Makefile

```diff
@@ -3,8 +3,6 @@ HELM_ARGS?=--wait --timeout 100s --create-namespace
 HELM_ACTION?=install
 
 free5gc:
-	phy_if=$(lshw -C network |grep -Fm 1 'logical name:' |awk -- '{print $NF}')
```

Comment:
If I'm not mistaken this was introduced by @PapaySail in order to run free5gc when physical interface name differs from configured by default

Reply:
Do we really need to remove these lines?